src/db-service.ts

```diff
@@ -0,0 +1,28 @@
+// global
+import { sql, DatabaseTransactionConnection as TrxHandler } from 'slonik';
+import { MemberRole } from './types';
+
+/**
+ * Database's first layer of abstraction for Admin DB services
+ */
+export class AdminService {
+
+  // QUESTION: Can a member have multiple roles? Do we expect to have more roles besides admin and normal user? 
```

Comment:
Indeed I think a member can have multiple roles, I think you can already return an array here!